xbmc/guilib/DirtyRegion.h

```diff
@@ -35,3 +35,10 @@ class CDirtyRegion : public CRect
 };
 
 typedef std::vector<CDirtyRegion> CDirtyRegionList;
+
+class CRenderingStats
+{
```

Comment:
@xhaggi I assume you are not referring to the existing ``ProcessInfo.h`` (used by VideoPlayer)? So we'll probably need a different name to avoid confusion.